Formula/qt@5.9.rb

```diff
@@ -7,17 +7,17 @@
 class QtAT59 < Formula
   desc "Cross-platform application and UI framework, Long-Term Support version"
   homepage "https://www.qt.io/"
-  url "https://download.qt.io/official_releases/qt/5.9/5.9.3/single/qt-everywhere-opensource-src-5.9.3.tar.xz"
-  mirror "https://www.mirrorservice.org/sites/download.qt-project.org/official_releases/qt/5.9/5.9.3/single/qt-everywhere-opensource-src-5.9.3.tar.xz"
-  sha256 "57acd8f03f830c2d7dc29fbe28aaa96781b2b9bdddce94196e6761a0f88c6046"
+  url "https://download.qt.io/official_releases/qt/5.9/5.9.7/single/qt-everywhere-opensource-src-5.9.7.tar.xz"
+  mirror "https://www.mirrorservice.org/sites/download.qt-project.org/official_releases/qt/5.9/5.9.7/single/qt-everywhere-opensource-src-5.9.7.tar.xz"
+  sha256 "70e617aeb1f9bbf84a12b8cf09b01ece"
```

Comment:
That's the MD5 :)

Think we're looking for `1c3852aa48b5a1310108382fb8f6185560cefc3802e81ecc099f4e62ee38516c`

Reply:
This shows just how tired I am 😭